File: app/src/main/java/com/neome/api/ent/base/dto/DtoEntFormExportExcel.java
// neome.ai API - do not change
//

package com.neome.api.ent.base.dto;

import com.neome.api.meta.base.Types.MetaIdForm;
import com.neome.api.meta.base.dto.DtoGridLayoutRefKey;
import com.neome.api.meta.base.dto.FormValueRaw;
import com.neome.api.meta.base.dto.GsonCto;

@SuppressWarnings("unused")
public class DtoEntFormExportExcel extends GsonCto
{
  public MetaIdForm formId;

  public FormValueRaw formValueRaw;

  public DtoGridLayoutRefKey[] layoutRefKeyList;
}

File: app/src/main/java/com/neome/api/ent/base/dto/DtoEntFormExportExcel.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.base.dto

import com.neome.api.meta.base.dto.DtoGridLayoutRefKey
import com.neome.api.meta.base.dto.FormValueRaw
import com.neome.api.meta.base.dto.GsonCto
import com.neome.api.meta.base.Types.MetaIdForm

interface DtoEntFormExportExcel : GsonCto
{
  val formId: MetaIdForm
  val formValueRaw: FormValueRaw
  val layoutRefKeyList: List<DtoGridLayoutRefKey>
}